firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/NewReleaseFetcher.java

```diff
@@ -195,6 +203,21 @@ private long getInstalledAppVersionCode(Context context) throws FirebaseAppDistr
     return PackageInfoCompat.getLongVersionCode(pInfo);
   }
 
+  private String getInstalledAppVersionName(Context context)
+      throws FirebaseAppDistributionException {
+    PackageInfo pInfo;
+    try {
+      pInfo = context.getPackageManager().getPackageInfo(context.getPackageName(), 0);
```

Comment:
Rather than initialize pInfo above, then set it here, and return it after the catch block, just return it here:
```
try {
  return context.getPackageManager().getPackageInfo(context.getPackageName(), 0).versionName;
} ...
```

Reply:
Good point